tests/validate_regexes.rs

```diff
@@ -0,0 +1,68 @@
+use regex::Regex;
+use serde::{Deserialize, Serialize};
+use std::fs;
+
+#[derive(Serialize, Deserialize, Debug, Default)]
+struct Example {
+    #[serde(rename(deserialize = "Valid"))]
+    valid: Vec<String>,
+    #[serde(rename(deserialize = "Invalid"), default)]
+    invalid: Vec<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Default)]
+struct TestCase {
+    #[serde(rename(deserialize = "Name"))]
+    name: String,
+    #[serde(rename(deserialize = "Regex"))]
+    regex: String,
+    plural_name: bool,
+    #[serde(rename(deserialize = "Description"))]
+    description: Option<String>,
+    #[serde(rename(deserialize = "Rarity"))]
+    rarity: f64,
+    #[serde(rename(deserialize = "URL"))]
+    url: Option<String>,
+    #[serde(rename(deserialize = "Tags"))]
+    tags: Vec<String>,
+    #[serde(rename(deserialize = "Examples"), default)]
+    examples: Example,
+}
+
+#[test]
+fn testcase_works() {
+    let path = format!("{}/src/data/regex.json", env!("CARGO_MANIFEST_DIR"));
+    let data: String = fs::read_to_string(path).unwrap();
+
+    let test_cases: Vec<TestCase> = serde_json::from_str(&data).unwrap();
+    for test_case in &test_cases {
+        if !test_case.examples.valid.is_empty() || !test_case.examples.invalid.is_empty() {
+            let re = Regex::new(&test_case.regex);
+            if let Ok(re) = re {
+                println!("{}", test_case.name);
```

Comment:
Instead of printing the regex which compile, print those which failed to compile and returned `Err`.

i.e. add a else block and in it put something like this
```rust
eprintln!("Failed to compile regex for {} due to {:#?}", test_case.name, re)
```